### ISSUE TITLE
Support for seconds

### ISSUE THREAD

**Issue by lancecarlson** [hlascelles/que-scheduler#190]
I noticed that fugit supports second based crons (ie "*/15 * * * * *"). I noticed that this will only run once a minute still. Were there any plans to support seconds?

**Comment by hlascelles** [comment]
Interesting question. Yes I suppose it is possible. The frequency is just stored as a constant [here](https://github.com/hlascelles/que-scheduler/blob/4b7c635c4bc7cbfd0d4eb9c441963ece805c58e4/lib/que/scheduler/scheduler_job.rb#L15).

Since the scheduler is self enqueueing, I think the best change would be to make it "run every minute, unless it knows it has to run faster". I chose one minute, as a reasonble default. Even if the scheduler "knows" its jobs only run every hour, it still needs to check more often as the dictionary may change (a new deploy) in that hour.

An alternative is to make a new deploy set the speed of the scheduler - but there be dragons. I can imagine an old and a new version getting into a battle setting the "speed".

So in summary, yes very possible. I will have a look and a think at the best/safest method.
